.happy/terraform/modules/sfn/main.tf

```diff
@@ -4,14 +4,15 @@ resource "aws_sfn_state_machine" "state_machine" {
 
   definition = <<EOF
```

Comment:
We should probably add a comment here indicating that this step function currently (temporarily) lives in 2 places:
* `single-cell-data-portal/.happy/terraform/modules/sfn/main.tf`
* `single-cell-infra/.happy/terraform/modules/sfn/main.tf`

and that to avoid confusion, they need to be identical until we merge them into one location.

Reply:
Good point, I will add it